docs/user/config-file/v2.rst

```diff
@@ -326,6 +326,15 @@ Node.js version to use.
    - ``19``
    - ``20``
 
+build.tools.ruby
+````````````````
+
+Ruby version to use.
+
+:Type: ``string``
+:Options:
+   - ``3.3``
```

Comment:
I think it's reasonable to only start with a single version. Should we have a small note saying you can ask support for additional versions? 🤷

Reply:
I think that people will naturally ask for more versions (and tools) as they have been doing 😄 without the need of a note 😅

Reply:
Ha, true.